### PR TITLE
[MFTF][WIP] Cover #708

### LINKED ISSUE #708: [MFTF] User views limited image information from the image grid in Media Gallery
# Description
Implement MFTF test to cover scenario: "User views limited image information from the image grid in Media Gallery" (story 41, #460)

#  Preconditions

1. Admin User is logged into Admin Panel
2. Adobe Stock Preview image exists in Media Gallery
3. Adobe Stock Licensed image exists in Media Gallery
4. User is signed into Adobe IMS

# Test steps

1. Open Media Gallery
2. User sees the following image attributes associated with all images:
  1. title - Asset's title
  2. type - jpg, gif, png ...
  3. dimensions
3. User doesn't see the `Unlicensed` label on licensed Adobe Stock image in Media Gallery
4. User sees the AdobeStock identification with `Unlicensed` on previewed Adobe Stock image in Media Gallery

# Additional details

Three tests should be created:
One in MediaGalleryUi module checking the attributes for uploaded image
Second in AdobeStockImageAdminUi module checking the label for Preview Adobe Stock images
Third in AdobeStockImageAdminUi module checking the label for Licensed Adobe Stock images

### PR DESCRIPTION
**Pull request by engcom-Charlie**
### Description (*)

- cover https://github.com/magento/adobe-stock-integration/issues/708
- fixed issue with absent `Licensed` label on image.

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#708: [MFTF] User views limited image information from the image grid in Media Gallery
2. ...

### Manual testing scenarios (*)

1. ...
2. ...

### CODE REVIEW

**Comment by engcom-Charlie** [comment]
@magento run all tests

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockImageInformationGridAssertActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockImageInformationGridAssertActionGroup.xml]: Please move this action group to MediaGalleryUi module

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockNewMediaGalleryDeleteImageActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAdobeStockNewMediaGalleryDeleteImageActionGroup">
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockNewMediaGalleryDeleteImageActionGroup.xml]: Please move this action group to MediaGalleryUi module

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PR @engcom-Charlie ! Please see my review comments

**Comment by engcom-Charlie** [comment]
the test will be passed after merge https://github.com/magento/adobe-stock-integration/pull/1129

**Comment by gabrieldagama** [comment]
Closing this PR since the test cases are also covered here:
https://github.com/magento/adobe-stock-integration/pull/1334

**Pull request closed without merging**